x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/chart_switch.tsx

```diff
@@ -106,11 +106,22 @@ function computeListHeight(list: SelectableEntry[], maxHeight: number): number {
   return Math.min(list.length * ENTRY_HEIGHT, maxHeight);
 }
 
+function safeFnCall<TReturn>(action: () => TReturn, defaultReturnValue: TReturn): TReturn {
```

Comment:
Again, I wonder about the silent error suppression here. What if we extended `safeFnCall` to accept a warning message to log if it returns the default, for example?